### ISSUE TITLE
Batch Normalization Layers not present in the converted Keras model

### ISSUE THREAD

**Issue by rahman-mdatiqur** [gmalivenko/pytorch2keras#135]
**Describe the bug**
No batch normalization layers present in the converted keras model.

**To Reproduce**
__Snippet of your code__

**Expected behavior**
All layers including the batch normalization layers should be present in the converted keras model
**Logs**
If applicable, add error message to help explain your problem.

**Environment (please complete the following information):**
 - OS: [e.g. iOS] Ubuntu-20.04
 - Python [e.g. Python 2, Python 3] - Python 3.8
 - Version [e.g. v0.1.11] - master

**Additional context**
When converting a pytorch model into keras_model, doing keras_model.summary() does not show the batch normalization layers.

**Comment by MaxPolak97** [comment]
Have you solved this issue?
